Gemfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Google LLC
```

Comment:
Used the file when tested the module locally, e.g. ran the commands `bundle exec kitchen test`, `bundle exec kitchen verify` etc. Is there a better way of local testing?
Removed the file, should it be added to `.gitignore`?

Reply:
It's left over from older versions of the module template. We don't need to worry about ignoring it because it shouldn't be created again in the future.

You should utilize the actions defined in the `Makefile` to run tests locally. `make docker_test_prepare` will create a test project and service account. `make docker_run` will put you in an interactive container where you can use `kitchen_do` to run `kitchen` with the new service account credentials.

Reply:
ok, got it